### PR TITLE
test: send test results to report portal dashboard

### PR DESCRIPTION
**Pull request by adeldhis2**
The purpose of this PR is to send the test results to a central test dashboard using https://reportportal.io/
This work is still in progress
I will share more details in the coming days and update the PR when ready to review

### CODE REVIEW

File: .github/workflows/e2e-dev.yml [hunk 1]
```diff
@@ -22,7 +22,6 @@ defaults:
 
 jobs:
     compute-dev-version:
-        if: "!github.event.push.repository.fork && (github.event.pull_request.draft == false || github.action == 'workflow_dispatch')"
```
Comment (Philip-Larsen-Donnelly) [inline .github/workflows/e2e-dev.yml:25]: Why have these conditions been removed?

File: .github/workflows/e2e-prod.yml [hunk 2]
```diff
@@ -40,7 +43,6 @@ defaults:
 
 jobs:
     compute-prod-versions:
-        if: "!github.event.push.repository.fork && (github.event.pull_request.draft == false || github.action == 'workflow_dispatch')"
```
Comment (Philip-Larsen-Donnelly) [inline .github/workflows/e2e-prod.yml:43]: Why have these conditions been removed?

File: .github/workflows/e2e-dev.yml (hunk 1 above)
Comment (adeldhis2) [reply .github/workflows/e2e-dev.yml:25]: I [removed this temporary](https://github.com/dhis2/data-visualizer-app/pull/2744/commits/9b2fcc88b003a5eb49b01e7443352fe42c481036) …I was not sure why the e2e test was skipped .. so with this change I was able to see the e2e tests running... I already asked the analytics team on slack about it

File: .github/workflows/e2e-prod.yml (hunk 2 above)
Comment (adeldhis2) [reply .github/workflows/e2e-prod.yml:43]: I [removed this temporary](https://github.com/dhis2/data-visualizer-app/pull/2744/commits/9b2fcc88b003a5eb49b01e7443352fe42c481036) …I was not sure why the e2e test was skipped .. so with this change I was able to see the e2e tests running... I already asked the analytics team on slack about it

**Pull request closed without merging**